src/coordination/include/coordination/coordinator_client.hpp

```diff
@@ -44,13 +45,20 @@ class CoordinatorClient {
   auto InstanceName() const -> std::string;
   auto SocketAddress() const -> std::string;
 
-  [[nodiscard]] auto SendPromoteReplicaToMainRpc(ReplicationClientsInfo replication_clients_info) const -> bool;
   [[nodiscard]] auto DemoteToReplica() const -> bool;
+  auto SendPromoteReplicaToMainRpc(const utils::UUID &uuid, ReplicationClientsInfo replication_clients_info) const
+      -> bool;
+
+
+  auto SendSwapMainUUIDRpc(const utils::UUID &uuid) const -> bool;
 
   auto ReplicationClientInfo() const -> ReplClientInfo;
 
+
   auto SetCallbacks(HealthCheckCallback succ_cb, HealthCheckCallback fail_cb) -> void;
 
+  auto RpcClient() -> rpc::Client & { return rpc_client_; }
```

Comment:
Since this method is used only to call `memgraph::replication_coordination_glue :SendSwapUUIDRpc` I would leave it as is in order not to duplicate that method, because I am calling it for replication and coordination